src/fastify-plugins/maps/static-maps.js

```diff
@@ -0,0 +1,284 @@
+import path from 'path'
```

Comment:
Nitpick:

```suggestion
// @ts-check
import path from 'path'
```

Reply:
same question about whether this is needed if already included by the tsconfig